docs/section11/Rule11-8.md

```diff
@@ -0,0 +1,54 @@
+# Service_Water_Heating - Rule 11-8
+**Schema Version:** 0.0.36  
+
+**Mandatory Rule:** TRUE
+
+**Rule ID:** 11-8
+
+**Rule Description:** "One system per building area type shall be modeled in the baseline."
+
+**Rule Assertion:** Options are PASS/FAIL
+
+**Appendix G Section Reference:** Table G3.1 #11, baseline column, a + b
+
+**Evaluation Context:** B-RMD each SHW type
+**Data Lookup:**   
+**Function Call:** 
+- **get_SHW_types_and_spaces**
+- **get_SHW_equipment_connected_to_use_type**  
+
+**Applicability Checks:**
+- check that the SHW type in the P_RMD has SHW loads - **is P_RMD appropriate for the applicability check, or should it be B_RMD?**  
+
+## Applicability Checks:
+- only projects with SHW for the SHW space type in the proposed model are expected to have a SHW system in the baseline model.  If there is no SHW in the proposed model, we assume no SHW system is needed.
+- use the function get_SHW_types_and_spaces to get a list of spaces for each SHW type: `shw_and_spaces_dict = get_SHW_types_and_spaces(P_RMD)`
+- look at each SHW bat: `for shw_bat in shw_and_spaces_dict:`
+  - look at each space: `for space_id in shw_and_spaces_dict[shw_bat]:`
+    - get the space using get_component_by_id: `space = get_component_by_id(P_RMD, space_id)`
+    - look for the ServiceWaterHeatingUse in the space.  If even one space has a ServiceWaterHeatingUse, continue to rule logic: `if len(space.service_water_heating_uses) > 0: CONTINUE TO RULE LOGIC`
```

Comment:
would this then keep repeating the logic below everytime there was a load encountered?

Reply:
yes